plugins/package-managers/pub/src/main/kotlin/Pub.kt

```diff
@@ -179,7 +179,9 @@ class Pub(
 
         logger.info { "Downloading flutter-$flutterVersion from $url... " }
         flutterInstallDir.safeMkdirs()
-        val flutterArchive = okHttpClient.downloadFile(url, flutterInstallDir).getOrThrow()
+        val flutterArchive = okHttpClient.downloadFile(url, flutterInstallDir).onFailure {
+            logger.warn { "Unable to download Flutter $flutterVersion from $url" }
```

Comment:
Nit: Missing dot at the end of the logged sentence.

Reply:
done